modules/social_features/social_topic/social_topic.module

```diff
@@ -364,7 +364,7 @@ function _social_topic_change_topic_icon_ajax(array $form, FormStateInterface $f
     '#type' => 'inline_template',
     '#template' => '<div class="icon-svg-wrapper" data-href="#' . $icon . '"><svg class="selected-svg"><use xlink:href="#' . $icon . '"></use></svg>{{ svg_icons|raw }}</div>',
     '#context' => [
-      'svg_icons' => file_get_contents(drupal_get_path('theme', 'socialbase') . '/assets/icons/icons.svg'),
+      'svg_icons' => file_get_contents(\Drupal::service('extension.list.module')->getPath('socialbase') . '/assets/icons/icons.svg'),
```

Comment:
The same, use `\Drupal::service('extension.list.theme')->getPath('socialbase');`.